internal/controller/standalone_pgadmin/pod_test.go

```diff
@@ -70,7 +70,7 @@ containers:
     valueFrom:
       secretKeyRef:
         key: password
-        name: pgadmin-standalone-pgadmin
+        name: pgadmin-
```

Comment:
I assume since the `cr` doesn't really exist, the UID is just an empty string?

Reply:
Yes, that's right. I think that's ok for these tests.